tasks/85_bet-validation/bet-validation.js

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line camelcase
+export function validate_bet(range, bet) {
+  const rangeOfNumbers = range;
```

Comment:
Copying the reference to `range` array. Change `rangeOfNumbers` array leads changing `range` array
Please take a look - [by reference vs. by value](https://stackoverflow.com/a/6605700)